java/tools/src/java/org/apache/orc/tools/FileDump.java

```diff
@@ -103,7 +103,7 @@ public static void main(Configuration conf, String[] args) throws Exception {
         rowIndexCols = null; // All the columns
       } else {
         String[] colStrs = cli.getOptionValue("r").split(",");
-        rowIndexCols = new ArrayList<Integer>(colStrs.length);
```

Comment:
The same mistake like https://github.com/apache/orc/pull/1181#discussion_r924633593